tests/README.md

```diff
@@ -0,0 +1,34 @@
+## Testing
```

Comment:
I think this is equivalent to the integration snapshoting logic in `router/tests`?